lib/standalone.mjs

```diff
@@ -117,6 +118,9 @@ export default async function standalone({
 
 	const formatterFunction = await getFormatter(stylelint);
 
+	const autofix =
+		(normalizeFixMode(stylelint._options.fix) ?? config?.fix === true) ? 'lax' : undefined;
```

Comment:
[suggestion] I wonder if the position of `()` is incorrect 🤔 The current `autofix` could be only `'lax'` or `undefined`, not `strict`. Is this intentional?

```suggestion
		normalizeFixMode(stylelint._options.fix) ?? (config?.fix === true ? 'lax' : undefined);
```

Reply:
Besides, if my concern is right, there might be no test cases for `'strict'` 🤔

Reply:
Thanks for your suggestion!
Actually, since `postcss.parse` is chosen in `lib/getPostcssResult.mjs` and a parse error results in `postcssResult` being `undefined`, there's no need for the separate `lax` and `strict` handling in `standalone.mjs`.

Fixe in this commit.
[refactor: remove unnecessary variables](https://github.com/stylelint/stylelint/pull/8106/commits/c28206ba586ddc8b0f8f2fdead689c6648293b85)